vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -581,14 +581,17 @@ public void emit(SectionRenderer sectionRenderer)
                     e ->
                         new Target() {
                           private final List<Pair<String, ConsumableResource>> consumables =
-                              Stream.concat(
+                              Stream.concat(Stream.concat(
                                       e.getValue().getConsumableResources().stream()
                                           .map(
                                               name ->
                                                   new Pair<>(name, consumableResources.get(name))),
                                       Stream.of(
                                           new Pair<String, ConsumableResource>(
-                                              "", maxInFlightPerWorkflow)))
+                                              "", maxInFlightPerWorkflow))),
+                                    Stream.of(
+                                            new Pair<String, ConsumableResource>(
+                                                "priority", priorityPerWorkflow)))
```

Comment:
how's it work that maxInFlight is "" but priority has a name?

Reply:
I'm not sure why it's "" either, other than it's simply never exposed so it doesn't matter?